reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+
+### Paused
+{%
+  include
+  figure
+  image_path="./resource-status-paused.png"
+  caption="Example of a resource with the Paused status"
+%}
+
+When a resource has a Paused status, it means Spinnaker is configured to continuously manage the resource but you've chosen to temporarily pause management. While management is paused Spinnaker won't check for drift from the declarative configuration, and it won't take any automatic actions. You can go to the Config view in your app to pause and resume management across the entire application.
+
+When you resume management, Spinnaker will check for drift from the declarative configuration. If a drift is detected, the status will change to [Diff](/reference/managed-delivery/resource-status/#diff). If no drift is detected, the status will change to [Happy](/reference/managed-delivery/resource-status/#unhappy).
+
+### Unknown
+{%
+  include
+  figure
+  image_path="./resource-status-unknown.png"
+  caption="Example of a resource with the Unknown status"
+%}
+
+When a resource has an Unknown status, it means Spinnaker was unable to determine the current status for some reason. Because the status is unknown, it's tough to tell by the status alone whether something is *wrong*, or just unexpected.
+
+After encountering this status, Spinnaker will continue trying to check the resource for its status and may eventually succeed. If it does, the resource will transition to whatever status is applicable at that time.
+
+If you see this status, it's probably best to investigate the current state of the resource manually and double check that nothing is broken. If you decide manual intervention is needed to remedy any issues you might discover, you can temporarily pause management on the Config view for your app.
+
+### Error
+{%
+  include
+  figure
+  image_path="./resource-status-error.png"
+  caption="Example of a resource with the Error status"
+%}
+
+When a resource has an Error status, it means something went wrong while trying to check the resource for drift from its declarative configuration. Because something went wrong, Spinnaker can't take automatic actions and drift from the declarative configuration won't be resolved. Some examples of problems that might lead to an error status:
+
+  - A problem retrieving or processing information about the current state of the resource
+  - A problem with the declarative configuration (invalid settings, etc.)
+  - A bug or issue with Spinnaker itself
+
+After an error, Spinnaker will continue trying to check the resource and may eventually succeed. If it does, the resource will transition to whatever status is applicable at that time.
+
+If you see this status, it's probably best to investigate the current state of the resource manually and double check that nothing is broken. If you decide manual intervention is needed to remedy any issues you might discover, you can temporarily pause management on the Config view for your app.
```

Comment:
How does one go about investigating current state and double checking nothing is broken? Should we include some troubleshooting guidance or is that out of scope for this PR?

Reply:
Yeah this is was I was eluding to with this somewhat defeatist comment in the PR description:
> Right now I'm providing some pointers to the task view and high level guidance like 'investigate the current state of the resource manually and double check that nothing is broken'. This is somewhat weak guidance because we don't have excellent debugging/observability tools yet, but once the history modal is built out I intend to add a similar section on using that view to debug issues with cross-linking from this page. 

IMO this is ok in the short term while folks at Netflix are the only people using it and know how to reach out to us, but soon we'll need to do better.